presto-tests/src/main/java/io/prestosql/tests/AbstractTestQueries.java

```diff
@@ -2395,6 +2395,17 @@ public void testShowCatalogsLike()
         assertEquals(result.getOnlyColumnAsSet(), ImmutableSet.of(getSession().getCatalog().get()));
     }
 
+    @Test
+    public void testShowCatalogsLikeWithEscape()
+    {
+        assertQueryFails("SHOW CATALOGS LIKE '%$_%' ESCAPE", "line 1:33: mismatched input '<EOF>'. Expecting: <string>");
```

Comment:
This should go in TestParserErrorHandling. Generally, we try to avoid engine-specific tests in AbstractTestQueries, since these run for every connector and can extend test run time considerably.